### ISSUE TITLE
Request: add support to search API to get results without "q" parameter

### ISSUE THREAD

**Issue by afernv10** [metmuseum/openaccess#34]
Hi!

I would like to request support for search API function. 

Right now this feature let you search only adding the _q_ parameter filled. However, in the Search Met Museum webpage ([https://www.metmuseum.org/art/collection/search]) you have more options to filter, only applying the filters and keeping empty the search bar you get all the results for **hasImages** and a selected department, for example. There arent other possibility to filter all the artworks with images, unless you get all artworks from a department and filter manually.

To sum up, I request for Search feature without _q_ parameter and getting all the results.

thanks a lot for your work!

**Comment by stephenhmarsh** [comment]
This would fix #38 @spencerk
